src/components/MagicCodeInput.js

```diff
@@ -348,8 +345,9 @@ function MagicCodeInput(props) {
                                 onKeyPress={onKeyPress}
                                 onPress={(event) => onPress(event, index)}
                                 onFocus={onFocus}
-                                caretHidden={isMobileSafari}
-                                inputStyle={[isMobileSafari ? styles.magicCodeInputTransparent : undefined]}
+                                selectionColor="transparent"
```

Comment:
Can you put that in a comment above this so that nobody ever does that and accidentally reintroduces that bug? 

Better yet, can we prevent this at a lower level if it breaks the paste function? Better to throw an error if anyone tries to add this.